### DISCUSSION TITLE
Send last seen timestamp to ha

### DISCUSSION THREAD

**Discussion by thecobra666** [zwave-js/zwave-js-ui#3496] [Ideas]
Hi, would it be possible to send the last seen timestamp to HA as an entity or sensor?

**Comment by robertsLando** [comment]
Hi @thecobra666 , sorry for the delay but I took a break during Christmas/new years's eve holidays. 

are you using mqtt discovery or zwave-js integration?

**Reply by thecobra666** [reply]
> Hi @thecobra666 , sorry for the delay but I took a break during Christmas/new years's eve holidays. 
> 
> are you using mqtt discovery or zwave-js integration? 

No problem. Happy new Year ! 

I'm using the integration.

**Reply by robertsLando** [reply]
Ok so I have no control over that, you should open the issue on home assistant core, I'm sorry :(

**Reply by thecobra666** [reply]
Done https://github.com/home-assistant/core/issues/106955 

Correction: https://community.home-assistant.io/t/zwavejs-missing-last-seen-entity/665702

Thanks anyway :)
